src/Resources/contao/dca/tl_social_feed.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+\Contao\System::loadLanguageFile('tl_social_feed');
```

Comment:
I'm not sure if this is the right solution - but otherwise there are some warnings for missing translation keys!